paasta_tools/setup_prometheus_adapter_config.py

```diff
@@ -288,6 +304,96 @@ def create_instance_uwsgi_scaling_rule(
     }
 
 
+def create_instance_piscina_scaling_rule(
+    service: str,
+    instance: str,
+    autoscaling_config: AutoscalingParamsDict,
+    paasta_cluster: str,
+) -> PrometheusAdapterRule:
+    """
+    Creates a Prometheus adapter rule config for a given service instance.
+    """
+    setpoint = autoscaling_config["setpoint"]
+    moving_average_window = autoscaling_config.get(
+        "moving_average_window_seconds", DEFAULT_UWSGI_AUTOSCALING_MOVING_AVERAGE_WINDOW
```

Comment:
imo, we should probably have a separate constant for the default piscina moving average window (even if it ends up being the same as the uwsgi one to start)